files/zh-cn/web/api/crypto/subtle/index.html

```diff
@@ -5,12 +5,18 @@
 ---
 <p>{{APIRef("Web Crypto API")}}</p>
 
+<p class="notecard secure">{{APIRef("Web Crypto API")}}{{SecureContext_header}}</p>
```

Comment:
Please replace the whole line with the macro.

Reply:
Done.